src/main/CMakeLists.txt

```diff
@@ -3,13 +3,14 @@ target_sources(clio PRIVATE impl/Build.cpp)
 
 target_link_libraries(clio PUBLIC clio_etl clio_feed clio_web clio_rpc)
 
-target_compile_features(clio PUBLIC cxx_std_20)
+target_compile_features(clio PUBLIC cxx_std_23)
 
 # Clio server
 add_executable(clio_server)
 target_sources(clio_server PRIVATE Main.cpp)
 target_link_libraries(clio_server PRIVATE clio)
 target_link_options(
-  clio_server PRIVATE $<$<AND:$<NOT:$<BOOL:${APPLE}>>,$<NOT:$<BOOL:${san}>>>:-static-libstdc++ -static-libgcc>
+  # For now let's assume that we only using libstdc++ under gcc
+  clio_server PRIVATE $<$<AND:$<BOOL:${is_gcc}>,$<NOT:$<BOOL:${san}>>>:-static-libstdc++ -static-libgcc>
```

Comment:
If possible maybe lets replace it with a regular if?

Reply:
There is a difference between this syntax and regular `if`. This one evaluates at `make` stage while the regular `if` evaluates at configuration stage. Having that said, I don't really see why this needed to be the strange syntax to start with. I think normal `if` could work here just as well. I was not sure however so decided to leave it as it was to avoid accidentally breaking something that i don't fully understand.